create-cluster.rb

```diff
@@ -267,6 +269,18 @@ def parse_options
       options[:vpc_name] = name
     end
 
+    opts.on("-g", "--no-gitcrypt", "Avoid the execution of git-crypt unlock (example: pipeline tools might do that for you)") do |name|
+      options[:gitcrypt_unlock] = false
+    end
+
+    opts.on("-i", "--no-integration-test", "Avoid the execution of git-crypt unlock (example: pipeline tools might do that for you)") do |name|
```

Comment:
The comment here is incorrect, it should be something like "Don't run integration tests after creating the cluster"

Reply:
Yeah sorry, copy-pasted from the above one. Fixed now.